wagtail/admin/templates/wagtailadmin/bulk_actions/select_all_checkbox_input.html

```diff
@@ -1,3 +1,2 @@
 {% load i18n %}
-{% trans "Select all" as aria_label %}
-<input {% if parent %}data-bulk-action-parent-id="{{ parent }}"{% endif %} data-bulk-action-select-all-checkbox type="checkbox" aria-label="{{ aria_label }}" />
+<input {% if parent %} data-bulk-action-parent-id="{{ parent }}" {% endif %} data-bulk-action-select-all-checkbox type="checkbox" aria-label="{{ _('Select all') }}"/>
```

Comment:
Does this one work? From what I can see in the Django docs, this format here (directly using the translated value) is not documented as supported. 

I can check when reviewing but thought I'd quickly comment.

Reply:
Oh. My bad. I'm not so sure. I'll do some looking around, though, but I'd still appreciate if you could also check while reviewing. Also, I've done a push to show what I've done with the documentation.

Reply:
Yep this works.

<img width="1303" alt="Screenshot 2023-10-26 at 7 36 48 am" src="https://github.com/wagtail/wagtail/assets/1396140/22bd1cb0-b5b1-4981-81cc-6607c8bf74ed">

However, we should adjust the accidental whitespace introduced.

```suggestion
<input {% if parent %}data-bulk-action-parent-id="{{ parent }}"{% endif %} data-bulk-action-select-all-checkbox type="checkbox" aria-label="{{ _('Select all') }}"/>
```

I can fix this when merging though.